spring-scaffold-cli/src/main/java/br/com/netodevel/command/template/TemplateHandler.java

```diff
@@ -3,10 +3,7 @@
 import br.com.generate.helpers.ScaffoldInfoHelper;
 import br.com.generator.core.GeneratorOptions;
 import br.com.templates_java.ComposeTemplate;
-import br.com.templates_java.config.jms_aws_sqs.EntryPointMessageGenerator;
-import br.com.templates_java.config.jms_aws_sqs.MessageListenerGenerator;
```

Comment:
Consider using a more specific exception type instead of the general `IOException` and `URISyntaxException`, to improve error handling and performance.